cpp/include/cudf/copying.hpp

```diff
@@ -341,12 +353,12 @@ void copy_range_in_place(column_view const& source,
  * If @p source and @p target refer to the same elements and the ranges overlap,
  * the behavior is undefined.
  *
- * @throws cudf::logic_error for invalid range (if
+ * @throws std::out_of_range for invalid range (if
  * @p source_begin > @p source_end, @p source_begin < 0,
  * @p source_begin >= @p source.size(), @p source_end > @p source.size(),
  * @p target_begin < 0, target_begin >= @p target.size(), or
  * @p target_begin + (@p source_end - @p source_begin) > @p target.size()).
```

Comment:
Can we define what constitutes a valid (or invalid) range above in the docstring, and then just say "throws if invalid range" here?

Reply:
Sure I can do that.